src/main/java/UMC_7th/Closit/domain/battle/service/BattleService/BattleCommandServiceImpl.java

```diff
@@ -194,8 +194,31 @@ private void validateChallengeBattle(Battle battle, Post post, Long userId, Long
         }
 
         // Status = INACTIVE일 경우에만 배틀 신청 가능
-        if (battle.getStatus().equals(Status.INACTIVE)) {
-            battle.challengeBattle(Status.PENDING);
+        if (battle.getBattleStatus().equals(BattleStatus.INACTIVE)) {
+            battle.challengeBattle();
+        }
+    }
+
+    private void validateVoteBattle(Battle battle, Long userId, BattleRequestDTO.VoteBattleDTO request) {
+        // 배틀이 아닌 게시글에 투표 불가능
+        if (!battle.getPost1().getId().equals(request.getPostId()) && !battle.getPost2().getId().equals(request.getPostId())) {
```

Comment:
```suggestion
        if(isInvalidBattlePost(battle, Long postId)
```
Example
```
private boolean isInvalidBattlePost(Battle battle, Long postId) {
    return !battle.getPost1().getId().equals(postId) && !battle.getPost2().getId().equals(postId);
}
```
메서드로 따로 묶는게 더 깔끔해 보일 것 같아요